apidoc/Titanium/Media/MusicPlayer.yml

```diff
@@ -92,13 +92,6 @@ events:
         playback state.
     since: '3.0.0'
 
-  - name: volumechange
-    summary: Fired when the volume changes.
-    description: |
-        Use the [volume](Titanium.Media.MusicPlayer.volume) property to set or determine
-        the current volume level for this player.
-    since: '3.0.0'
-
```

Comment:
Sorry, I was initially wrong here: The `volumechange` event is still fired once the volume changes, different to the `volume` property that is the read-only version to receive the volume at any time.

Reply:
Ah cool, looking at the code this is specific to the MusicPlayer right? I'll just remove the description for this entry, or should we note that it's not possible to retrieve the volume value?

Reply:
But it is possible to receive the `volume` using the getter on a music-player instance.